doc/manual_experimental.md

```diff
@@ -26,9 +26,8 @@ oneself.
 Void type
 =========
 
-The `void` type denotes the absence of any type. Parameters of
-type `void` are treated as non-existent, `void` as a return type means that
-the procedure does not return a value:
+The `void` type denotes the absence of any value, i.e. it is the type that contains no values. Consequently, no value can be provided for parameters of
+type `void`, and no value can be returned from a function with return type `void`:
```

Comment:
Hm, while I get what you mean by the addition of "is the type that contains no values", stricly speaking a type never *contains* a value, but rather, a value *is of* a type.

Was there a specific scenario or misunderstanding that prompted this change?

Reply:
Right, I was implicitly thinking of types as the sets of values that are of the type. Then how about "it is the type that no value has" or "no value can ever be of type `void`"?

I did not have any specific scenario for prompting this change, I just read over this part and experienced friction when reading. So, I thought I might help out by removing the friction...